Gemfile.lock

```diff
@@ -0,0 +1,143 @@
+    ffi (1.9.10)
+    gemoji (2.1.0)
+    github-pages (40)
+      RedCloth (= 4.2.9)
+      github-pages-health-check (= 0.5.3)
+      jekyll (= 2.4.0)
+      jekyll-coffeescript (= 1.0.1)
+      jekyll-feed (= 0.3.1)
+      jekyll-gist (= 1.4.0)
+      jekyll-mentions (= 0.2.1)
+      jekyll-paginate (= 1.1.0)
+      jekyll-redirect-from (= 0.8.0)
+      jekyll-sass-converter (= 1.3.0)
+      jekyll-sitemap (= 0.9.0)
+      jemoji (= 0.5.0)
+      kramdown (= 1.9.0)
+      liquid (= 2.6.2)
+      maruku (= 0.7.0)
+      mercenary (~> 0.3)
+      pygments.rb (= 0.6.3)
+      rdiscount (= 2.1.8)
+      redcarpet (= 3.3.3)
+      terminal-table (~> 1.4)
+    github-pages-health-check (0.5.3)
+      addressable (~> 2.3)
+      net-dns (~> 0.8)
+      public_suffix (~> 1.4)
+      typhoeus (~> 0.7)
+    html-pipeline (1.9.0)
+      activesupport (>= 2)
+      nokogiri (~> 1.4)
+    i18n (0.7.0)
+    jekyll (2.4.0)
+      classifier-reborn (~> 2.0)
+      colorator (~> 0.1)
+      jekyll-coffeescript (~> 1.0)
+      jekyll-gist (~> 1.0)
+      jekyll-paginate (~> 1.0)
+      jekyll-sass-converter (~> 1.0)
+      jekyll-watch (~> 1.1)
+      kramdown (~> 1.3)
+      liquid (~> 2.6.1)
+      mercenary (~> 0.3.3)
+      pygments.rb (~> 0.6.0)
+      redcarpet (~> 3.1)
+      safe_yaml (~> 1.0)
+      toml (~> 0.1.0)
+    jekyll-coffeescript (1.0.1)
+      coffee-script (~> 2.2)
+    jekyll-feed (0.3.1)
+    jekyll-gist (1.4.0)
+      octokit (~> 4.2)
+    jekyll-mentions (0.2.1)
+      html-pipeline (~> 1.9.0)
+      jekyll (~> 2.0)
+    jekyll-paginate (1.1.0)
+    jekyll-redirect-from (0.8.0)
+      jekyll (>= 2.0)
+    jekyll-sass-converter (1.3.0)
+      sass (~> 3.2)
+    jekyll-sitemap (0.9.0)
+    jekyll-watch (1.3.0)
+      listen (~> 3.0)
+    jemoji (0.5.0)
+      gemoji (~> 2.0)
+      html-pipeline (~> 1.9)
+      jekyll (>= 2.0)
+    json (1.8.3)
+    kramdown (1.9.0)
+    liquid (2.6.2)
+    listen (3.0.5)
+      rb-fsevent (>= 0.9.3)
+      rb-inotify (>= 0.9)
+    maruku (0.7.0)
+    mercenary (0.3.5)
+    mini_portile2 (2.0.0)
+    minitest (5.8.3)
+    multipart-post (2.0.0)
+    net-dns (0.8.0)
+    nokogiri (1.6.7.1)
+      mini_portile2 (~> 2.0.0.rc2)
+    octokit (4.2.0)
+      sawyer (~> 0.6.0, >= 0.5.3)
+    parslet (1.5.0)
+      blankslate (~> 2.0)
+    posix-spawn (0.3.11)
+    public_suffix (1.5.3)
+    pygments.rb (0.6.3)
+      posix-spawn (~> 0.3.6)
+      yajl-ruby (~> 1.2.0)
+    rb-fsevent (0.9.7)
+    rb-inotify (0.9.5)
+      ffi (>= 0.5.0)
+    rdiscount (2.1.8)
+    redcarpet (3.3.3)
+    safe_yaml (1.0.4)
+    sass (3.4.20)
+    sawyer (0.6.0)
+      addressable (~> 2.3.5)
+      faraday (~> 0.8, < 0.10)
+    terminal-table (1.5.2)
+    thread_safe (0.3.5)
+    toml (0.1.2)
+      parslet (~> 1.5.0)
+    typhoeus (0.8.0)
+      ethon (>= 0.8.0)
+    tzinfo (1.2.2)
+      thread_safe (~> 0.1)
+    yajl-ruby (1.2.1)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  github-pages
+
+BUNDLED WITH
+   1.11.2
```

Comment:
That's actually strange... I'll try to auto-generate it again and see if there's a bug upstream.